src/components/TeamMemberCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
You can delete this line as beginning from recent versions you don't need to import React in your component.